src/epoll.rs

```diff
@@ -7,6 +7,39 @@ use std::os::unix::io::RawFd;
 use vmm_sys_util::epoll::{ControlOperation, Epoll, EpollEvent};
 
 use super::{Errno, Error, EventOps, Result, SubscriberId};
+use std::ops::{Deref, DerefMut};
+
+/// Collection of events corresponding to the `epoll` ready list.
+pub(crate) struct ReadyEvents(Vec<EpollEvent>);
```

Comment:
The PR and commit message refer to a `ReadyList` but the new struct is `ReadyEvents`, is this the same thing or is there some missing commit that adds `ReadyList` too?